### DISCUSSION TITLE
How could I fix some paths to adapt to client changes?

### DISCUSSION THREAD

**Discussion by flazerain** [PyVRP/PyVRP#592] [Q&A] [answered]
if clients change when part of the path planning has already been executed, , can I re-solve by fixing part of the solution(executed)?

**Comment by N-Wouda** [comment, marked as answer]
Hi @flazerain, we don't currently support a way to model the sorts of client-vehicle restrictions you'd need for re-optimisation. That is planned, and the issue is tracked in #329! In particular, a start to #329 will be made when we complete #475. That is scheduled to be part of the upcoming release. Future releases might make the modelling nicer, but that is not yet planned.
